firmware/src/src/sense.cpp

```diff
@@ -433,13 +433,33 @@ void calculate_Thread()
         int tltch = trkset.tiltCh();
         int rllch = trkset.rollCh();
         int panch = trkset.panCh();
+        int alertch = trkset.alertCh();
         if(tltch > 0)
             channel_data[tltch - 1] = trpOutputEnabled == true ? tiltout_ui : trkset.Tlt_cnt();
         if(rllch > 0)
             channel_data[rllch - 1] = trpOutputEnabled == true ? rollout_ui : trkset.Rll_cnt();
         if(panch > 0)
             channel_data[panch - 1] = trpOutputEnabled == true ? panout_ui : trkset.Pan_cnt();
 
+        // TODO what is trpOutputEnabled?
```

Comment:
It disables the Tilt/Roll and Pan on long button press. Was a feature requested to make it more like the Fatshark Trinity where the output would only enable/disable on long button presses.